INTERNAL.md

```diff
@@ -0,0 +1,115 @@
+# Internal documentation
+
+This document has information relevant to the maintainers of this repository.
+End-user technical documentation should live within the charts themselves or in the `gh-pages` branch.
+
+## Package signing
+
+Release automation signs Helm chart releases using a GPG key stored in a [GitHub Actions secret](https://github.com/grafana/helm-charts/settings/secrets/actions).
+Only repository administrators have permissions to update secrets.
+Release automation uses the `GPG_KEY_BASE64` secret to sign the packages.
+
+The `GPG_KEY_BASE64` is a base64 encoded GPG key.
+It expires on 2023-03-30.
+
+Grafana Labs employees may access the private key using from the company 1password.
+
+Grafana Labs employees can extend the expiry of the private key using the `gpg` command line tool.
+
+1. Write the contents of the `Loki Helm GPG Key` 1password secret into a file.
+
+1. Import the key into a separate keyring.
+
+   ```console
+   $ gpg --no-default-keyring --keyring ~/.gnupg/helm-charts.gpg --import <PATH TO GPG KEY>
+   gpg: key 7054A9559D3CFB0B: public key "Grafana Loki <loki-team@googlegroups.com>" imported
+   gpg: key 7054A9559D3CFB0B: secret key imported
+   gpg: Total number processed: 1
+   gpg:               imported: 1
+   gpg:       secret keys read: 1
+   gpg:  secret keys unchanged: 1
+   ```
+
+1. Run the `gpg` interactive prompt to edit the 'Grafana Loki' key.
+
+   ```console
+   $ gpg --no-default-keyring --keyring ~/.gnupg/helm-charts.gpg --edit-key 'Grafana Loki'
+   gpg (GnuPG) 2.3.4; Copyright (C) 2021 Free Software Foundation, Inc.
+   This is free software: you are free to change and redistribute it.
+   There is NO WARRANTY, to the extent permitted by law.
+
+   Secret key is available.
+
+   sec  rsa2048/7054A9559D3CFB0B
+        created: 2020-03-29  expires: 2023-03-30  usage: SC
+        trust: unknown       validity: ultimate
+   ssb  rsa2048/4A6B2462555868C7
+        created: 2020-03-29  expires: 2023-03-30  usage: E
+   [ultimate] (1). Grafana Loki <loki-team@googlegroups.com>
+
+   gpg>
+   ```
+
+1. At the `gpg` interactive prompt, run the `expire` command and follow the prompt to extend expiry.
```

Comment:
isn't this redundant?

Reply:
Which bit exactly is redundant?

Perhaps the following is more clear?
```suggestion
1. At the `gpg` interactive prompt, run the `expire` command and follow the subsequent prompt to extend expiry.
```

Reply:
The whole section, since for the next step you have "key 1" and "expire" again

Reply:
Ah the next steps also extend the expiry of the subkey.

Reply:
thank you, now got it